lighthouse-core/audits/multi-check-audit.js

```diff
@@ -66,7 +65,7 @@ class MultiCheckAudit extends Audit {
   /**
    * @param {LH.Artifacts} artifacts
    * @param {LH.Audit.Context} context
-   * @return {Promise<{failures: Array<string>, warnings?: Array<string>, manifestValues?: LH.Artifacts.ManifestValues}>}
+   * @return {Promise<{failures: Array<string>, manifestValues?: LH.Artifacts.ManifestValues}>}
```

Comment:
`webapp-install-banner` was the only one using `warnings` for start_url warnings, so no longer needed